infrastructure/quick-deploy/aws/storage/outputs.tf

```diff
@@ -22,7 +22,8 @@ output "storage_endpoint_url" {
       kms_key_id            = module.s3_os[0].kms_key_id
     } : null
     mongodb = {
-      url = module.mongodb.url
+      url                = module.mongodb.url
+      number_of_replicas = var.mongodb.replicas_number
```

Comment:
You need some consistency in the naming.

Reply:
"replicas_number" and "number_of-replicas" refer to the same value. But, I have used "replicas_number" where it is used as input variable and "number_of_replicas" where it is used as output variable. Having two names is not required, I did it to avoid the confusion.